CRM.Library.Services/HomeService/BookService.cs

```diff
@@ -1,21 +1,32 @@
-﻿using DAL.Interface;
+﻿using DAL.Interfaces;
 using DAL.Models;
 using System.Collections.Generic;
 
 namespace CRMLibrary.Services.HomeService
 {
     public class BookService : IBookService
     {
-        public BookService(IGenericRepository<Book> repositoryBook)
+        public BookService(IBookRepository bookRepository)
```

Comment:
why did you change IGenericRepository<Book> to IBookRepository?

Reply:
I thought that IGenericRepository consists of main methods for all repositories, other interfaces IBookRepository, IUserRepository which inherited from IGenericRepository, will be specific methods for own repository.
![image](https://user-images.githubusercontent.com/62796151/126264504-40bd88c3-6508-4e7c-8141-8a129b84151c.png)